### ISSUE TITLE
Introduce ActionPlatform class to be used as range of actionPlatform

### ISSUE THREAD

**Issue by betehess** [schemaorg/schemaorg#1150]
`actionPlatform` currently accepts `Text` and `URL`. It is not clear which values are allowed.

Here are a few things we could do:
* create a new `ActionPlatform` class, "A high level platform where an `Action` can be performed."
* make `actionPlatform` accept a `ActionPlatform`
* add a few `ActionPlatform` individuals

[Google's documentation](https://developers.google.com/structured-data/actions/#target) currently defines the following platforms:

* http://schema.org/DesktopWebPlatform: Works on desktop web browsers.
* http://schema.org/MobileWebPlatform: Works on mobile web browsers.
* http://schema.org/AndroidPlatform: Works on Android native app.
* http://schema.org/IOSPlatform: Works on iOS native app.

I am still not sure if it's a good idea to add too many individuals in Schema.org for a case like that. Unlike `DayOfWeek`, we know we'll have to add new things and that approach is not extensible. Maybe we could try to add this information in Wikidata directly instead? Here are a few candidates:

* http://www.wikidata.org/entity/Q466
* http://www.wikidata.org/entity/Q1043805
* http://www.wikidata.org/entity/Q94
* http://www.wikidata.org/entity/Q48493

After we have annotated those entities in Wikidata, we could provide a SPARQL query that lists all the entities being a `ActionPlatform`.

Or maybe that's not a good idea? :-)

**Comment by danbri** [comment]
I advised against finalizing an enumeration at this stage (some notes in #1121) but explorations towards that are certainly useful... /cc @vholland

**Comment by betehess** [comment]
I totally agree re: "I advised against finalizing an enumeration at this stage".

I somehow missed #1121, sorry for that. Should I close this issue? Or should we discuss possible paths to be explored here? Not sure how to proceed on that one :-)

**Comment by stewardson** [comment]
Did this issue get resolved? 

Attempting to add ReserveAction schema as Google recommends here https://developers.google.com/search/docs/data-types/local-business

However Google's structured testing tool https://search.google.com/structured-data/testing-tool/ reports the value as invalid:

https://schema.org/DesktopWebPlatform (The attribute actionPlatform has an invalid value.)

https://schema.org/DesktopWebPlatform currently 404's and the enumeration values suggested here #773 e.g. DesktopWebPlatform is also an invalid value.

Or is this issue closed and therefore this particular schema not supported by Google?

**Comment by danbri** [comment]
@betehess - still interested?

**Comment by danbri** [comment]
Closing for #3057 to follow up on the broad proposal
